app/plan.html

```diff
@@ -5,9 +5,9 @@ <h3>Plan Your Event</h3>
 <br>
 
 <ul>
-    <big>Submit your <a href="https://form.jotform.com/222676794136163">raza
+    <big>Submit your <a href="PLANNING_URL">raza
         request</a>. For questions, please contact <a
-     href="mailto:secretary&#64;sfjamaat.org">our jamaat secretary</a>.
+     href="mailto:SECRETARY_EMAIL">our jamaat secretary</a>.
```

Comment:
nit: Under 12 Umoor framework. This should now be Umoor FMB coordinator.

Reply:
Okay. So let's make title of the role also a template variable. Sound good?